src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -554,11 +554,13 @@ public static function create(array $attributes)
 	 */
 	public static function forceCreate(array $attributes)
 	{
-		static::unguard();
```

Comment:
I would keep the original code and just add `if (static::$unguarded) return  static::create($attributes);`